chainlit_app/services/feedback_service.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+import requests
+
+API_URL = os.getenv("API_URL", "http://localhost:8000")
+
+
+def submit_feedback(user_id: str, feedback: str) -> dict:
+    """
+    Sends feedback from a user to the feedback API.
+
+    Args:
+        user_id (str): The user's unique ID.
+        feedback (str): The feedback content.
```

Comment:
I think the feedback endpoint need to be updated. It is not clear what the feedback is for.